projects/pycryptodome/project.yaml

```diff
@@ -0,0 +1,3 @@
+homepage: "https://www.pycryptodome.org/"
+language: c
+primary_contact: "helderijs@gmail.com"
```

Comment:
do we have any confirmation from this maintainer? Did you talk to the over email or maybe can tag them on GitHub?

Reply:
Not yet, I'll reach out to them in a bit.

Reply:
Thanks @rjotwani ! https://github.com/Legrandin/pycryptodome/issues/441 looks good to me